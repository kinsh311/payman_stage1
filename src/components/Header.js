import React from 'react';
import {NavLink} from 'react-router-dom';
const Header = () => (
    <header>
        <h1>Expensify test</h1>
        <br />
        <NavLink to="/home" activeClassName = "is-active" exact={true}>Home</NavLink><br /> {/*NavLink has more props and styling features*/}
        <NavLink to="/create" activeClassName = "is-active">Add expense</NavLink><br />

    </header>
)
export default Header