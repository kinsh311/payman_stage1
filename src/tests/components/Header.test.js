
import React from 'react';
import Header from '../../components/Header';
import { shallow } from 'enzyme';

test('rendering header', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();

    // expect(wrapper.find('h1').text()).toBe('Expensify test');
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
})