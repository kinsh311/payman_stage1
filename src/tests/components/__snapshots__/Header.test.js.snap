// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering header 1`] = `
<header>
  <h1>
    Expensify test
  </h1>
  <br />
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/home"
  >
    Home
  </NavLink>
  <br />
   
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Add expense
  </NavLink>
  <br />
</header>
`;
